packages/medusa/src/services/cart.ts

```diff
@@ -648,7 +641,7 @@ class CartService extends TransactionBaseService {
             .create({
               ...lineItem,
               has_shipping: false,
-              cart_id: cartId,
+              cart_id: cart.id,
             })
         }
 
```

Comment:
> Wdyt of avoiding joining all items in addLineItem like I have done here: [develop...fix/serialized-add-to-cart#diff-b1b00df3e31608f5099498d5dc7dfc5f92032809e1e16cbd7b1e9a85304c8edbR578](https://github.com/medusajs/medusa/compare/develop...fix/serialized-add-to-cart#diff-b1b00df3e31608f5099498d5dc7dfc5f92032809e1e16cbd7b1e9a85304c8edbR578)
> 
> Curious if this would be more performant in cases where the cart holds many items, despite the extra db queries?

I can do it as it always good to reduce to the necessary data, in terms of perf it should not change that much but it is always good to get 😀

Reply:
Don’t know why the answer has been put there sorry 😂